manual/parts/glossary.tex

```diff
@@ -0,0 +1,116 @@
+%
+% Copyright 2014, General Dynamics C4 Systems
+%
+% SPDX-License-Identifier: GPL-2.0-only
+%
+
+\newglossaryentry{asid}{
+    name=ASID,
+    description={Address Space Identifier. ASIDs are associated with page directories (PDs), and define the virtual address space of a thread. Multiple threads may be in the same address space}
+}
+
+\newglossaryentry{badge}{
+    name=Badge,
+    description={A badge is a piece of extra information stored in a capability. It can be used by applications to identify caps previously handed out to clients}
```

Comment:
```suggestion
    description={A badge is a piece of extra information stored in a capability, mostly used in endpoint and notification capabilities. It can be used by applications to identify caps previously handed out to clients}
```

Reply:
"with" or "for" seems more appropriate than "in". Otherwise a good improvement. Any other use cases for badges are truly obscure and I can't think of anything useful, other than perhaps limiting further copying or revocation.

Reply:
Yup, happy with "for"

Reply:
I made the change using:
description={A badge is a piece of extra information stored in a capability, mostly used for endpoint and notification capabilities. It can be used by applications to identify caps previously handed out to clients}